### ISSUE TITLE
Subtracting bias in photospline/I3SplineTable even if derivatives are available (Trac #2236)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix4#2228]
After evaluating photosplines a bias must only be subtracted if no derivatives are available but it is always done.
I am going to commit a fix for that soon.

<details>
 <summary><em>Migrated from <a href="https://code.icecube.wisc.edu/projects/icecube/ticket/2236">https://code.icecube.wisc.edu/projects/icecube/ticket/2236</a>, reported by mhieronymusand owned by mhieronymus</em></summary>
<p>

```json
{
    "status": "closed",
    "changetime": "2019-02-04T17:52:51",
    "_ts": "1549302771596052",
    "description": "After evaluating photosplines a bias must only be subtracted if no derivatives are available but it is always done.\nI am going to commit a fix for that soon.",
    "reporter": "mhieronymus",
    "cc": "",
    "resolution": "fixed",
    "time": "2019-01-31T08:17:04",
    "component": "combo reconstruction",
    "summary": "Subtracting bias in photospline/I3SplineTable even if derivatives are available",
    "priority": "normal",
    "keywords": "Bug I3SplineTable",
    "milestone": "",
    "owner": "mhieronymus",
    "type": "defect"
}
```

</p>
</details>

**Comment by icecube-trac** [comment]
Trac update at 20190131T08:29:41: 
* mhieronymus commented:

> In changeset 171057:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="171057"
> Fix #2228 Subtract bias only if derivatives are involved.
> ```


* mhieronymus changed status from "new" to "closed"
* mhieronymus changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190204T17:52:51: nwhitehorn commented:

> In changeset 171112:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="171112"
> Fix #2228 Subtract bias only if derivatives are involved.
> ```
